app/src/main/res/values/strings.xml

```diff
@@ -957,7 +957,8 @@ A level counts as a roof level when its windows are in the roof. Subsequently, r
 
     <string name="quest_camp_drinking_water_title">Is drinking water freely available to campers here?</string>
     <string name="quest_camp_shower_title">Are there showers here?</string>
-    <string name="quest_camp_power_supply_title">Is there any way for customers to charge their phone, laptop etc. here?</string>
+    <string name="quest_camp_power_supply_title">Are there any power supply sockets available for guests?</string>
+    <string name="quest_camp_power_supply_hint">This can be sockets at the pitches, guest usable sockets in a service building, sockets for phone/laptop charging, etc.</string>
```

Comment:
```suggestion
    <string name="quest_camp_power_supply_hint">This could be sockets directly at the campsite, sockets in a service building accessible by guests, a dedicated charging station for phones, laptops, etc.</string>
```

Reply:
Hmmmm, _"directly at the **campsite**"_ sounds quite confusing/**problematic** to me. I know SC nontranslated texts are in US English, but while _"campsite"_ is indeed more popular in US, it seems to be regional there too?

If we want to avoid _"camping pitch"_ (more popular in British English) we should IMO find some alternative way to say what we want (because _"campsite"_ in BE means what _"campground"_ does in US -- i.e. the whole fenced area, and not the individual pitch, which is highly confusing --  see https://en.wikipedia.org/wiki/Campsite for details)

(another related issue is that _"power socket"_ is AFAICT more British term, preferred US term apparently being _"electrical outlet"_).

Any native English speakers around? Help!

Reply:
Oh, I didn't know that. I actually meant the British English version. I.e. some sockets somewhere on the camping area. (And you meant sockets directly for each parcel?)

Reply:
Right, electrical outlet makes sense.

https://books.google.com/ngrams/graph?content=power+outlet%2C+electrical+outlet%2C+power+socket%2C+electrical+socket&year_start=1900&year_end=2019&corpus=en-US&smoothing=3

Power outlet is also fine. (Just asked in the #language channel on slack-us)

Reply:
People in the #language channel find "power outlet" to be more common. It's also shorter!

Reply:
OK I've tried to incorporate that; how does it look now? Let's suggest new edits if needed.